javascript/javascript1/week4/tic_tac.js

```diff
@@ -0,0 +1,49 @@
+//Tic Tac Toe
+
+function getRenderedGame(position){
+    const renderedGame=[];
+    for(let i=0; i<position.length; i++){
+        const star='*'+position[i].join('*')+'*';// applying join method
+        renderedGame.push(star);
+      
+    } 
+    return '*******\n'+renderedGame.join('\n')+'\n*******';
+    }
+// console with example
+    const position=[
+        ['x', 'o', ' '],
+        [' ', 'o', ' '],
+        [' ', 'o', 'x']
+    ];
+console.log(getRenderedGame(position));
+ 
+
+//Determining a winner
+function getGameinfo(position){
+    const gameInfo={};
+    for(let i=0; i<position.length; i++){
```

Comment:
you are missing some cases like this one:

  ['x', 'x', 'x '],
  [' ', ' ' , ' '],
  [' ', ' ', ' ']
position[0][1], position[0][2], position[0][3]
  [' ' ,  ' , ' '],
  [' x', 'x ' , 'x'],
  [' ', ' ', ' ']
position[1][1], position[1][2], position[1][3]

Can you think of more?